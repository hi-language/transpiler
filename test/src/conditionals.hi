a: 10
b: 20
max: (a > b) ? { a } : { b }
_(max)

status: "ok"
(status == "ok") ? { _("Status is OK") }

grade: (max > 15) ? { "A" } : { "B" }
_(grade)
