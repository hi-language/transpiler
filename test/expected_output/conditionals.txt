20
Status is OK
A
